database/migrations/2021_08_17_124500_create_events_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateEventsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('events', function (Blueprint $table) {
+            $table->id();
+            $table->integer('country_id');
+            $table->string('title');
+            $table->string('place');
+            $table->string('price');
+            $table->string('explanation');
```

Comment:
データ入稿してみたら、stringじゃ入りきらなかったのでtext型に変更お願いします！
```suggestion
            $table->text('explanation');
```